### DISCUSSION TITLE
GET request for a specific article returns "text" filed where intro text and full text are concatenated together

### DISCUSSION THREAD

**Discussion by MayaSima** [joomla/joomla-cms#46995] [2. Cold Features]
I need to automatically process fulltext from multiple articles. I pull the article, process fulltext, and then send it back via PATCH.
I noticed that "text" field in the GET request response contains both introtext and fulltext concatenated with  "\r\n \r\n" like so:

`<p>Intro Text.</p>

<p>First Paragraph of full text.</p>`

As a result, I do the processing, publish the result back as a "fulltext", which results in intro text being copied to the full text and being duplicated on the updated page.

### Steps to reproduce the issue
`GET request to {{base_url}}/{{base_path}}/content/articles/{{article_id}}`
parse the returned payload and retrieve "text" attribute
process
PATCH request wit hthe updated fulltext

### Expected result
"fulltext" and "introtext" should be kept as separate fields in the response from GET request to {{base_url}}/{{base_path}}/content/articles/{{article_id}} endpoint

Not sure if "text" field is needed at all

### Actual result
I receive "text" field with concatenated "introtext" + "fulltext" with no `<hr id="system-readmore">` separator in between in the response from the `GET request to {{base_url}}/{{base_path}}/content/articles/{{article_id}}` endpoint

### System information (as much as possible)
Joomla version 4.3.3

**Comment by alikon** [comment]
please test #44566

**Comment by alikon** [comment]
as pr #44566  has been closed i'm repoening this issue
